plugins/woocommerce/tests/e2e-pw/tests/api-tests/customers/customers-crud.test.js

```diff
@@ -166,10 +165,41 @@ test.describe(
 		test.describe( 'Create a customer', () => {
 			test( 'can create a customer', async ( { request } ) => {
 				// call API to create a customer
+				const username = `john.doe.${ Date.now() }`;
```

Comment:
Use unique data. When running multiple times we either had failures or false negative results.